### PR TITLE
KHADRI-19092024-858-PM

### PR DESCRIPTION
**Pull request by SalukulaMahendra**

### CODE REVIEW

**Review by manvithagali52 — APPROVED** [review]

**Review by thasmiyas39 — APPROVED** [review]

**Review by thasmiyas39 — APPROVED** [review]

File: src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java [hunk 1]
```diff
@@ -2,16 +2,20 @@
...
 import java.sql.Connection;
 import java.sql.PreparedStatement;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 import com.khadri.jdbc.prepare.connection.JdbcConnectionUtil;
 import com.khadri.jdbc.prepare.data.movie.Movie;
 
 public class MovieInsertDao {
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java]: change name from MovieInsertDao to MovieDao

File: src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java [hunk 2]
```diff
@@ -22,8 +26,24 @@ public void movieInsertData(Movie movie) throws Exception {
...
+			System.out.println("MovieInsertDao Exception occours:" + exc);
+		}
+	}
+
+	public void movieSelectData() {
+
+		try {
+			Connection con = JdbcConnectionUtil.getConnection();
+			stmt = con.createStatement();
+			ResultSet resultSet = stmt.executeQuery("Select * from movie");
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java]: why titled case for query ???

File: src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java (hunk 1 above)
Comment (SalukulaMahendra) [reply src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java]: changed MovieInsertDao to MovieDao

File: src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java (hunk 2 above)
Comment (SalukulaMahendra) [reply src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java]: changed titled case

**Pull request merged**